addons/bitmap-copy/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Bitmap images copying",
-  "description": "Allows you to actually copy a bitmap image from the paint editor, so that you can paste it in other websites or software.",
+  "description": "Allows you to actually copy a bitmap image from the costume editor, so that you can paste it in other websites or software.",
```

Comment:
"Actually" does mean something here, since you can already copy and paste bitmap images inside the same project. Maybe we should mention this copies to the "global system clipboard" or something like that

Reply:
That's not what actually means. Actually sounds like it's a brand new super cool thing (its not even the coolest addon we have) and it sounds unprofessional.

Reply:
I'd use something like *Allows you to copy a bitmap image from the costume editor into the system clipboard*